### ISSUE TITLE
tgbl-coin is not weighted

### ISSUE THREAD

**Issue by hcagri** [shenyangHuang/TGB#100]
Hi, 

First, thank you for your hard work and sharing this benchmark. I’m using the tgbl-coin dataset and noticed that all the edge weights are set to 0. Your paper states it as a weighted graph, but I could not find any edge weights.

Thank you

**Comment by shenyangHuang** [comment]
Hi, 

Thank you for raising this issue and your kind words. 
We will fix it in the upcoming patches. 
The dataset is indeed weighted, the original edgelist of the dataset can be accessed [here](https://object-arbutus.cloud.computecanada.ca/tgb/tgbl-coin-v2.zip). 
The issue is created when we moved to separate weight from edge message, previously weight would be appended to message while currently it is separated for clarity but not utilized in the PyG data. 

The current solution would be to access in this way: 
```
from tgb.linkproppred.dataset import LinkPropPredDataset

DATA = "tgbl-coin"
dataset = LinkPropPredDataset(name=DATA, root="datasets", preprocess=True)
data = dataset.full_data  
data["w"]
```
This will provide you with the weights to all edges in `full_data`

Best,
Andy

**Comment by hcagri** [comment]
Thanks for the detailed response!

**Comment by shenyangHuang** [comment]
Problem fixed in issue #102
